turbinia/api/utils.py

```diff
@@ -28,6 +28,19 @@
 
 log = logging.getLogger(__name__)
 
+ATTACHMENT_RESPONSE = {
```

Comment:
You can leave this out and specify it in the openapi.yaml file which is how we specify content-type for other repsonses as well (eg json).

```
responses:
        '200':
          content:
            application/octet-stream:
              schema:
                type: string
                format: binary
```